kubernetes/deployment-production.tmpl

```diff
@@ -351,13 +349,11 @@ spec:
           image: zooniverse/panoptes:__IMAGE_TAG__
           resources:
             requests:
-              memory: "4000Mi"
+              memory: "500Mi"
```

Comment:
> It doesn't cause extra pressure to burst past this number?

Yes - It will add to RAM pressure on the underlying node (depending on the sidekiq workloads) but it will help with pod scheduling across the available nodes (less RAM requirement so should have more node RAM to pack the pods into). In turn this should mean less node autoscaling as well (as nodes only scaled when a pod can't schedule). 

> Sidekiq is sitting at 467Mi right now, which seems kind of uncomfortably close unless the goal is to request just past standard usage and let it creep past occasionally. Totally nonblocking, just curious and wanting to understand where you feel like these numbers should be relative to one another.

500Mi is the request which is the starting point for a pod to be scheduled so provides some headroom for a pod to grow it's RAM use. 

The 500 value is a trade off between packing pods on the available nodes (lower is better) and the best value for avoiding OOM errors on the service (node runs out of available RAM). This is something to keep an eye on will the how pods use RAM over time and how much we have available on the nodes.